modules/ROOT/pages/_partials/task-view-governed-apis-for-rulesets.adoc

```diff
@@ -0,0 +1,29 @@
+// Used in docs-exchange/asset-details.adoc and docs-api-governance-documentation/view-conformance-status-in-exchange.adoc
+
+You can view the list of centrally-governed APIs to which a ruleset applies. Depending on the access you have to the API assets in Exchange, you may see only the APIs that are conformant to the ruleset or you may see APIs that are conformant and not conformant to the ruleset. For example, asset or organization administrators or contributors can see APIs in both conformance statuses. 
+
+Developers can use the *Governed APIs* list to help them improve the quality of the rulesets and APIs they develop as follows: 
+
+* When their APIs fail a ruleset, they can see which other APIs are successfully conforming to the ruleset so that they can fix their APIs.
+* When they are discovering available rulesets to use for their API, they can see which APIs are using a specific ruleset to evaluate how helpful it is.
```

Comment:
probably instead of using, we could say "passing" or are conformant, to avoid the permission discussion again :)

Reply:
Fixed as suggested in Slack